src/code/actions.js

```diff
@@ -268,16 +268,20 @@ function getEarnedCoinString(state) {
   return ["~ALERT.NEW_PIECE_OF_COIN", scoreString];
 }
 
+function getVisibleGenes(state, routeSpec) {
+  const visibleGenes = AuthoringUtils.getChallengeDefinition(state.authoring, routeSpec).visibleGenes;
```

Comment:
Eli and I have talked about this. I even wrote a PT story for talking with Rob about what ITS actually wants/needs and cleaning up the code. I'd be game for changing `visibleGenes` to `staticGenes` or something else if it makes things clearer, but I think what Rob says the ITS actually wants should be factored into any naming discussion. The name(s) should make sense from an authoring standpoint as well as a logging standpoint.